api/src/main/java/com/jmanday/api/login/LoginApi.kt

```diff
@@ -0,0 +1,8 @@
+package com.jmanday.api.login
+
+import com.jmanday.client.RetrofitClient
+
+class LoginApi {
+
+    val service = RetrofitClient("https://sparky.pl-coding.com:8081/").retrofit.create(LoginService::class.java)
```

Comment:
Rather than having one API instance per call, I'd bundle them per feature (in this case for authentication), so an `AuthApi` will safe you a lot of boilerplate while keeping the code clear. This can contain login, registration, access token stuff and so on